src/ar/tuktukcinema/src/eu/kanade/tachiyomi/animeextension/ar/tuktukcinema/Tuktukcinema.kt

```diff
@@ -31,7 +31,7 @@ class Tuktukcinema : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     override val name = "توك توك سينما"
 
-    override val baseUrl = "https://w.tuktukcinema.net"
+    override val baseUrl = "https://w.tuktukcinema.tv/"
```

Comment:
The base URL is always without a trailing slash.

```suggestion
    override val baseUrl = "https://w.tuktukcinema.tv"
```